lib/build.js

```diff
@@ -14,7 +14,9 @@ var fs = require('graceful-fs')
   , mkdirp = require('mkdirp')
   , exec = require('child_process').exec
   , processRelease = require('./process-release')
-  , win = process.platform == 'win32'
+  , win = process.platform === 'win32'
+if (win)
```

Comment:
You don't need this if you already injected `config.variables.msbuild_path` (which is smart)